src/supernode/src/test/java/com/alibaba/dragonfly/supernode/service/impl/ProgressServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package com.alibaba.dragonfly.supernode.service.impl;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+
+import com.alibaba.dragonfly.supernode.common.domain.gc.GcMeta;
+import com.alibaba.dragonfly.supernode.common.enumeration.PeerPieceStatus;
+import com.alibaba.dragonfly.supernode.common.view.ResultInfo;
+import com.alibaba.dragonfly.supernode.repository.PeerRepository;
+import com.alibaba.dragonfly.supernode.repository.PeerTaskRepository;
+import com.alibaba.dragonfly.supernode.repository.ProgressRepository;
+import com.alibaba.dragonfly.supernode.repository.TaskRepository;
+import com.alibaba.dragonfly.supernode.service.impl.PeerServiceImpl;
+import com.alibaba.dragonfly.supernode.service.impl.PeerTaskServiceImpl;
+import com.alibaba.dragonfly.supernode.service.impl.ProgressServiceImpl;
+import com.alibaba.dragonfly.supernode.service.impl.TaskServiceImpl;
+import com.diffblue.deeptestutils.Reflector;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.api.mockito.expectation.PowerMockitoStubber;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.BitSet;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+
+@PowerMockIgnore({"javax.management.*"})
+@RunWith(PowerMockRunner.class)
+public class ProgressServiceImplTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: ProgressServiceImpl */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 458 branch to line 459
+   *  - conditional line 461 branch to line 464
+   */
+
```

Comment:
Discard - This would possibly be better in a newer version

Reply:
discard

Reply:
Removed.